client/src/app/components/Header/MobileMenu/mobile-menu.css

```diff
@@ -0,0 +1,172 @@
+/*Hamburger styling */
+
+input,
+textarea,
+button,
+select,
+nav,
+a {
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  display: none;
+  width: 75px;
+  height: 75px;
+  outline: none;
+}
+
+.bar {
+  width: 100%;
+  height: 5px;
+  background-color: #312a31;
+}
+
+nav:hover {
+  cursor: pointer;
+}
+
+#menu {
+  height: 50%;
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+#bar-1.bar-1-animate {
+  animation: bar-1 0.3s;
+  animation-fill-mode: forwards;
+}
+#bar-1.bar-1-animate-rev {
```

Comment:
It's better to be explicit and use 'reverse' rather than 'rev', which might not be so obvious for others.